cli/install/install.go

```diff
@@ -884,18 +1037,41 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 			logFile)
 	} else {
 		log.Infof("Downloading tarantool...")
-		err = downloadRepo(search.GitRepoTarantool, tarVersion, path, logFile, installCtx.verbose)
+		if versionFound {
```

Comment:
After merging, please create a "code health" task to remove multiple code duplications from `install<Program>` functions.

Reply:
Ok